src/packages/documents/documents/entity-bulk-actions/manifests.ts

```diff
@@ -11,28 +11,30 @@ const entityActions: Array<ManifestEntityBulkAction> = [
 		alias: 'Umb.EntityBulkAction.Document.Move',
 		name: 'Move Document Entity Bulk Action',
 		weight: 10,
+		api: UmbDocumentMoveEntityBulkAction,
 		meta: {
 			label: 'Move',
 			repositoryAlias: DOCUMENT_REPOSITORY_ALIAS,
-			api: UmbDocumentMoveEntityBulkAction,
-		},
-		conditions: {
-			entityType,
 		},
+		conditions: [{
+			alias: 'Umb.Condition.CollectionEntityType',
+			match: entityType,
+		}],
 	},
 	{
 		type: 'entityBulkAction',
 		alias: 'Umb.EntityBulkAction.Document.Copy',
 		name: 'Copy Document Entity Bulk Action',
 		weight: 9,
+		api: UmbDocumentCopyEntityBulkAction,
 		meta: {
 			label: 'Copy',
 			repositoryAlias: DOCUMENT_REPOSITORY_ALIAS,
-			api: UmbDocumentCopyEntityBulkAction,
-		},
-		conditions: {
-			entityType,
 		},
+		conditions: [{
+			alias: 'Umb.Condition.CollectionEntityType',
```

Comment:
Is it "dangerous" to call this condition "collectionEntityType", if the bulk actions can be used in other places than collections in the future?

The entity actions are so generic, that they in theory can be used anywhere.